ProjFS.Mac/PrjFS.xcodeproj/project.pbxproj

```diff
@@ -242,6 +277,7 @@
 				4391F88F21E42AC40008103C /* Info.plist */,
 				4391F89321E42AC40008103C /* KauthHandler.cpp */,
 				4391F8A421E42AC50008103C /* KauthHandler.hpp */,
+				F5E39C8121F11556006D65C2 /* KauthHandlerTestable.hpp */,
```

Comment:
Can `KauthHandlerTestable.hpp` be moved into a unit testing specific folder (possibly `PrjFSKextsTests`)?

Reply:
We could, but it honestly seemed a bit easier to find/understand having it as a single file along-side the others.  The file name is pretty self explanatory.  I could honestly go either way here.

Reply:
Now that we're sticking with including the header in the .cpp I think it's fine to leave it side by side.